examples/constellations/degenerate-empty.json

```diff
@@ -1,4 +1,4 @@
 {
-    "name" : "empty-for-internal-tests-only",
+    "name" : "Empty (for internal tests only)",
     "roles" : {}
```

Comment:
Let's leave a test comment here.

Reply:
I saw it. I think you're correct about the right way to do reviews here.